stress/src/logs.rs

```diff
@@ -3,7 +3,7 @@
     OS: Ubuntu 22.04.3 LTS (5.15.146.1-microsoft-standard-WSL2)
     Hardware: AMD EPYC 7763 64-Core Processor - 2.44 GHz, 16vCPUs,
     RAM: 64.0 GB
-    53 M/sec
+    69 M/sec
```

Comment:
Does this mean that more memory is consumed in benchmark?

Reply:
@lalitb maybe use https://github.com/open-telemetry/opentelemetry-rust/tree/main/stress#feature-flags and compare the before and after results, so we know what is the impact (for the stress test!). (We don't need to update this file with that number, but can leave it as a comment in this PR for easy reference in the future)

Reply:
> Does this mean that more memory is consumed in benchmark?

This is a measure of throughput, 69 Million events/sec

Reply:
Good point, update the PR description with memory stats.